versioned_docs/version-5/migration.mdx

```diff
@@ -695,6 +695,145 @@ actor.subscribe((state) => {
 
 ## Actors
 
+### Use actor logic creators for `invoke.src` instead of functions
+
+:::breakingchange
+
+Breaking change
+
+:::
+
+The available actor logic creators are:
+
+- `createMachine`
+- `fromPromise`
+- `fromObservable`
+- `fromEventObservable`
+- `fromTransition`
+- `fromCallback`
+
+See [Actors](actors.mdx) for more information.
+
+<Tabs>
+<TabItem value="v5" label="XState v5">
+
+```ts
+// ✅
+import { fromPromise, createMachine } from 'xstate';
+
+const machine = createMachine({
+  invoke: {
+    src: fromPromise(async ({ input }) => {
+      const data = await getData(input.userId);
+      // ...
+      return data;
+    }),
+    input: ({ context, event }) => ({
+      userId: context.userId,
+    }),
+  },
+});
+```
+
+</TabItem>
+
+<TabItem value="v4" label="XState v4">
+
+```ts
+// ❌ DEPRECATED
```

Comment:
```suggestion
```
It might be confusing why this is deprecated since it's the same import as in the v5 example.

Reply:
@mellson I can see how it looks like this import line is deprecated in the code diff. Do you think though that it's clear enough from looking at the tabbed view that the entire tab's contents are what'd deprecated since it's in contrast to the Check mark icon for the v5 tab? That convention is used across all tabbed content as far as I know CC @laurakalbag 
![CleanShot 2023-06-05 at 11 52 15](https://github.com/statelyai/docs/assets/279239/e8314b04-7ee5-4460-b18b-41910c16132e)

Reply:
Maybe if it said `// Example **X** DEPRECATED` it might be more obvious that it's labeling the whole code block?